lib/sassc/script/value_conversion.rb

```diff
@@ -0,0 +1,44 @@
+module SassC
+  module Script
+    module ValueConversion
+      class Unsupported < StandardError; end
```

Comment:
unfortunately, i don't think we can rescue from "A SassC error", because this also rescues any exceptions that occur during execution of a custom function.  (Which may not descend from our base error)